guvnor-structure/guvnor-structure-backend/src/main/java/org/guvnor/structure/backend/repositories/RepositoryServiceImpl.java

```diff
@@ -263,6 +281,8 @@ public void removeRepository( final String alias ) {
                     if ( repository.getAlias().equals( alias ) ) {
                         organizationalUnitService.removeRepository( ou,
                                                                     repository );
+                        String metadata = ou.getName() + "/" + alias;
```

Comment:
Won't this fail if the ```finalAlias``` was originally created with [this](https://github.com/droolsjbpm/guvnor/pull/361/files#diff-0d90a1d4d7ccbb7bb35bb56aad51f4feR245)?